packages/openneuro-app/src/scripts/refactor_2021/containers/front-page-container.tsx

```diff
@@ -9,32 +12,144 @@ import {
   Infographic,
   RecentData,
   TopViewed,
+  Loading,
 } from '@openneuro/components'
 
-const FrontPageContainer: React.FC = () => {
-  const responsive = {
-    superLargeDesktop: {
-      // the naming can be any, depends on you.
-      breakpoint: { max: 4000, min: 3000 },
-      items: 5,
-      slidesToSlide: 5,
-    },
-    desktop: {
-      breakpoint: { max: 3000, min: 1024 },
-      items: 3,
-      slidesToSlide: 3,
-    },
-    tablet: {
-      breakpoint: { max: 1024, min: 464 },
-      items: 2,
-      slidesToSlide: 2,
-    },
-    mobile: {
-      breakpoint: { max: 464, min: 0 },
-      items: 1,
-      slidesToSlide: 1,
-    },
+const TOP_VIEWED = gql`
+  query top_viewed_datasets {
+    datasets(
+      first: 5
+      orderBy: { views: descending }
+      filterBy: { public: true }
+    ) {
+      edges {
+        node {
+          id
+          analytics {
+            views
+          }
+          latestSnapshot {
+            tag
+            summary {
+              modalities
+            }
+            description {
+              Name
+            }
+          }
+        }
+      }
+    }
+  }
+`
+
+const RECENTLY_PUBLISHED = gql`
+  query recently_published_datasets {
+    datasets(
+      first: 5
```

Comment:
I think it'd be nice if we could use a number that would allow all the layouts to have a full set of items on the last slide. We could do that by getting 30 items, or by setting `items` and `slidesToSlide` on superLargeDesktop to 4 and setting `first` in the queries to 12.